QuickOSM/test/test_configuration_panel.py

```diff
@@ -14,16 +15,36 @@
 
 class TestUiMainWindow(unittest.TestCase):
 
+    def setUp(self) -> None:
+        pass
+
+    def tearDown(self) -> None:
+        dialog = Dialog(get_iface())
+        dialog.combo_default_overpass.setCurrentIndex(0)
+        dialog.save_config_overpass.click()
+        dialog.combo_default_nominatim.setCurrentIndex(0)
+        dialog.save_config_nominatim.click()
```

Comment:
Can we use the `setUp` function and just use `save_settings(OVERPASS[0])` ?
Using the "dialog" (which is a "heavy" object, a full dialog in memory) to only change the value of the QgsSettings looks a little bit "overkill" ;-)